### DISCUSSION TITLE
DSS Integrating R Quarto & Python Language for Dynamic Reporting

### DISCUSSION THREAD

**Discussion by dimasrepo** [teamalgoritma/community#922] [Q&A] [answered]
Dear Team Algoritma, 

Pertanyaan :

1. Kenapa ada eror pada output dari syntax inline codenya ?
Input
![image](https://github.com/user-attachments/assets/2f637ffe-73ed-45d3-8ced-d83ac9d76308)
Output
![image](https://github.com/user-attachments/assets/53dc2bf3-81fb-4351-852b-0fa3b3aafa86) 

2. Bagaimana cara merubah page layout quarto ? Sudah di chek di [Quarto Page Layout](https://quarto.org/docs/output-formats/page-layout.html)
Layout awal
![image](https://github.com/user-attachments/assets/8aa775f7-6a1b-48e3-96eb-fd744b52bc07)
Layout target
![image](https://github.com/user-attachments/assets/af88ce9a-0ae7-445e-8fd5-89df3ecb01c6)

Mohon bantuannya, terimakasih

Best Regards

Dimas

**Comment by VictorNugraha** [comment]
Halo Pak @dimasrepo,

Untuk pertanyaan pertama, boleh minta tolong untuk diperlihatkan chunk yang menyimpan variabel C1? Untuk mengecek variabel tersebut terisi dengan nilai apa saja? Kami cukup curiga kenapa bisa muncul tipe data, karena pada variabel tersebut juga tersimpan pengecekan tipe datanya

Untuk pertanyaan kedua, kode yang digunakan seperti apa? Apakah bisa minta tolong untuk dilampirkan juga? Pada bagian material utama materi DSS, bisa merujuk ke bagian Mengatur Tampilan Laporan, dimana sudah ada contoh dari pengaturan Mulci Column Layout.
![image](https://github.com/user-attachments/assets/74115863-523a-4ebc-b297-647fdd51970f)

Mohon bantuan untuk konfirmasinya.
Terima kasih

**Comment by dimasrepo** [comment]
Dear Mas Victor,

Terimakasih untuk responnya.

1. Berikut ini penjelasannya C1, variabel saja mas di assign ulang ke code lain agar lebih pendek.
variable assign
![image](https://github.com/user-attachments/assets/258e2bb3-e817-470b-b73e-49dde192a0c3)
table
![image](https://github.com/user-attachments/assets/e3ef0049-c8a9-48bb-97ac-bacf88e443d5)

2. Oh iyaa sudah saya coba semalam format examplenya tidak berhasil, Saat ini saya sudah berhasil menggunakan format berikut mas
![image](https://github.com/user-attachments/assets/d6dc1c3f-dbd1-443d-ab35-849a2f0ef5ec)

Terima kasih

**Reply by VictorNugraha** [reply, marked as answer]
Siap terima kasih untuk balasannya, Pak @dimasrepo 

Untuk kendala pertama yang masih dihadapi, boleh Bapak coba pastikan kembali untuk variabel C1 diisi dengan hasil subseting, dan dicek dulu pada hasil chunknya, karena dari code yang dituliskan sudah benar harusnya. Dan untuk penulisannya, bisa coba dibuat lebih simple saja dengan `C1 = round(abs(df.iloc[.., ...]), 2)`

Tapi pada hasilnya cukup aneh karena menunjukan keseluruhan nilai dari tabel/data, mungkin saja nilai variabel C1 tertimpa dengan nilai sebelumnnya sehingga menjadi
![image](https://github.com/user-attachments/assets/72198536-fa48-43fe-a068-d8ef572c2593)

Sedangkan untuk pertanyaan yang kedua, sudah berhasil ya Pak. Nice!

Silahkan coba dicek kembali untuk jawaban yang pertama, jika masih terkendala silahkan untuk menghubungi kami kembali.

**Comment by dimasrepo** [comment]
Sudah mas sudah bisa saat re input kembali variabelnya.

Terimakasih untuk bantuannya
